arangodb-net-standard/AqlFunctionApi/Models/GetQueryRulesResponseItem.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+
+namespace ArangoDBNetStandard.AqlFunctionApi.Models
+{
+    /// <summary>
+    /// Represents a response item returned by 
+    /// <see cref="IAqlFunctionApiClient.GetQueryRulesAsync(System.Threading.CancellationToken)"/>
+    /// </summary>
+    public class GetQueryRulesResponseItem
+    {
+        public string Name { get; set; }
+        public GetQueryRulesResponseFlags Flags { get; set; }
```

Comment:
What about simplifying the type name to `QueryRuleFlags`?

Actually `GetQueryRulesResponseItem` could also be simplified to `QueryRule` 🤔 If you consider that the response if a `List<QueryRule>` . This is what we've done for `GetCurrentlyRunningAqlQueriesAsync` which returns a `List<RunningAqlQuery>`.

What do you think?

Reply:
Good idea. I've made the change.